deployment/v3/external/data-archive/install.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+# Installs data-archive
+## Usage: ./install.sh [kubeconfig]
+
+if [ $# -ge 1 ] ; then
+  export KUBECONFIG=$1
+fi
+
+NS=data-archive
+CHART_VERSION=1.0.0
+
+echo Create $NS namespace
+kubectl create ns $NS
+
+function installing_data-archive() {
+  read -p "Is values.yaml for data-archive chart set correctly as part of Pre-requisites?(Y/n) " yn;
+  if [ "$yn" != "Y" ]; then
+    echo "ERROR: values.yaml not set correctly; EXITING;";
+    exit 1;
+  fi
+  helm repo update
+
+  read -p "Please enter the time(hr) to run the cronjob every day (time: 0-23) : " time
+  if [ -z "$time" ]; then
+     echo "ERROR: Time cannot be empty; EXITING;";
+     exit 1;
+  fi
+  if ! [ $time -eq $time ] 2>/dev/null; then
+     echo "ERROR: Time $time is not a number; EXITING;";
+     exit 1;
+  fi
+  if [ $time -gt 23 ] || [ $time -lt 0 ] ; then
+     echo "ERROR: Time should be in range ( 0-23 ); EXITING;";
+     exit 1;
+  fi
+
+  read -p "Is archival running for sandbox installation? (Y/N): " archival_running
```

Comment:
Add prompt to tell user about action which will be performed for sandbox installation and other